openhands/server/listen.py

```diff
@@ -798,4 +798,9 @@ def github_callback(auth_code: AuthCode):
     )
 
 
+@app.get('/config.json')
+def get_config():
+    return FileResponse('./openhands/server/static/config.json')
```

Comment:
If we put this in the frontend static files (e.g. where we put image assets) we should get this part for free